waspc/examples/todoApp/main.wasp

```diff
@@ -39,6 +39,10 @@ app todoApp {
         },
       },
     },
+    hooks: {
```

Comment:
`auth.hooks.postSignup` and `auth.hooks.postLogin` is something users define in their Wasp file.

Do we want to put it somewhere else? `auth.postLogin`?

Reply:
We didn't have `hooks` so far, so maybe I would skip that, just go with `auth.postLogin`, as you said. What do you like better?